### PR TITLE
Make this plugin installable

### PR DESCRIPTION
**Pull request by garethky**
This PR adds an install script and instructions for setting this up with Moonraker for automatic updates.

I want to use this extension but I don't want it to disrupt my Klipper install or Moonraker's automatic updates of Klipper. This will also help other tool changer owners that want to try this out. And it will also help you get updates and bug fixes out to users when you make them.

### CODE REVIEW

**Comment by TypQxQ** [comment]
Thank you! 
Looks good and I wanted to test this early today and then work has come between. I will test it and merge as soon as possible!
/Andrei

**Comment by garethky** [comment]
Please do test it. The first version I posted did have a bug with the `ln` command. With the fix:

```
ls -la
...
-rw-r--r-- 1 pi pi 11247 Feb 25 13:01 tmc_uart.py
-rw-r--r-- 1 pi pi 10793 Mar  5 13:57 tmc_uart.pyc
lrwxrwxrwx 1 pi pi    41 Mar  9 21:07 toolgroup.py -> /home/pi/Klipper_ToolChanger/toolgroup.py
lrwxrwxrwx 1 pi pi    40 Mar  9 21:07 toollock.py -> /home/pi/Klipper_ToolChanger/toollock.py
lrwxrwxrwx 1 pi pi    36 Mar  9 21:07 tool.py -> /home/pi/Klipper_ToolChanger/tool.py
-rw-r--r-- 1 pi pi  6568 Feb 25 13:01 tsl1401cl_filament_width_sensor.py
-rw-r--r-- 1 pi pi  4594 Feb 25 13:01 tuning_tower.py
...
```

I also had to install it my form and branch, so I couldn't follow that part of the directions exactly.

To get this to work correctly with Moonraker's update manager you have to create a tag in your git repo that follows the `v1.2.3` format: https://moonraker.readthedocs.io/en/latest/configuration/#client-configuration

> Git repos must have at least one tag for Moonraker to identify its version. The tag may be lightweight or annotated. The tag must be in semantic version format, vX.Y.Z, where X, Y, and Z are all unsigned integer values. For example, a repos first tag might be v0.0.1.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,41 @@ parameter to specify another tool.
...
+type: git_repo
+path: ~/Klipper_ToolChanger
+origin: https://github.com/TypQxQ/Klipper_ToolChanger.git
+install_script: install.sh
+is_system_service: False
+```
+
+3) Run the `install.sh` script
+```
+./install.sh
```
Comment (TypQxQ) [inline README.md:70]: Maybe change this to ``~/Klipper_ToolChanger/install.sh``

**Review by TypQxQ — APPROVED** [review]
Great work!

**Pull request merged**